code/game/objects/items/weapons/storage/pouch.dm

```diff
@@ -0,0 +1,454 @@
+/obj/item/storage/pouch
+	name = "abstract pouch"
+	desc = "The physical manifestation of a concept of a pouch. Woah."
+	icon = 'icons/obj/clothing/pouches.dmi'
+	icon_state = "small_drop"
+	w_class = ITEMSIZE_LARGE //does not fit in backpack
+	max_w_class = ITEMSIZE_SMALL
+	slot_flags = SLOT_POCKET
+	storage_slots = 1
```

Comment:
I feel this is really "gamey" and makes no sense. 

Pouches hold a few small items, and you can put them in your pocket... but you can't put them in your backpack? So how big are they exactly? Belts are bigger than these surely, but can be put in your backpack.